qubes-i3status

```diff
@@ -127,8 +129,14 @@ status_disk() {
 }
 
 status_volume() {
-    local volume=$(awk '/%/ {gsub(/[\[\]]/,""); print $4}' <(amixer sget Master))
-    local playback=$(awk '/%/ {gsub(/[\[\]]/,""); print $6}' <(amixer sget Master))
+    local volcmd=''
+    if amixer sget Master 2>/dev/null; then
+        volcmd=$(amixer sget Master)
+    else
+        volcmd=$(amixer -c 1 sget Master)
```

Comment:
```suggestion
    if ! volcmd=$(amixer sget Master 2>/dev/null); then
        volcmd=$(amixer -c 1 sget Master)
```
Avoid double amixer sget call. Not tested.

Reply:
@marmarek Thank you committed your suggestion in https://github.com/QubesOS/qubes-desktop-linux-i3-settings-qubes/pull/12/commits/1d2391f83877474a717bd3ff70fd27217e4f419a .